mgmt/src/processor/proc.rs

```diff
@@ -190,6 +212,28 @@ pub async fn apply_gw_config(config: &mut GwConfig, frrmi: &mut FrrMi) -> Config
 
     /* apply in frr: need to render and call frr-reload */
     if let Some(internal) = &config.internal {
+        debug!("internal information base {internal:?}");
+        let mut rib: RequiredInformationBase = match internal.try_into() {
+            Ok(rib) => rib,
+            Err(err) => {
+                error!("{err}");
+                return Err(ConfigError::FailureApply);
+            }
+        };
+
+        debug!("required information base: {rib:?}");
+
+        let manager = VpcManager::<RequiredInformationBase>::new(netlink);
+        let mut required_passes = 0;
+        while !manager
+            .reconcile(&mut rib, &manager.observe().await.unwrap())
+            .await
+        {
+            required_passes += 1;
+            if required_passes >= 300 {
+                panic!("took more than 300 passes to reconcile interfaces")
```

Comment:
Why a `panic!` and not an error?

Reply:
API stability during integration.  Will fix in a follow on PR.